packages/launchpad/src/store/config.ts

```diff
@@ -0,0 +1,72 @@
+import { reactive, readonly, inject, App } from 'vue'
+import { Bundler } from '../utils/bundler'
+import { Framework } from '../utils/frameworks'
+import { TestingType } from '../utils/testingTypes'
+import { StoreApp } from './app'
+
+type ComponentSetup = {
```

Comment:
I think we have a convention to use `interface` over `type` for object definitions (like below in `StateConfig`)